tests/fetching/test_gitrepo.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+import fetching
+
+import repoutils
+
+
+# TODO: Convert to setup / teardown
+def test_numpy():
```

Comment:
The test function names do not describe their functionality enough. Use something like if_numpy_repository_is_fetched_repository_is...

Reply:
Good point :+1: